napari_animation/utils.py

```diff
@@ -71,7 +63,7 @@ def keys_to_list(input_dict):
             yield [key]
 
 
-def make_thumbnail(image: np.ndarray, shape=(32, 32, 4)) -> np.ndarray:
+def make_thumbnail(image: np.ndarray, shape=(30, 30, 4)) -> np.ndarray:
```

Comment:
lol :)  just making the individual keyframes a _little_ bit shorter in height.
but this would actually be a great feature to add: something that controls the size of each keyframe in the list